modules/manage/pages/manage-xdcr/create-xdcr-replication.adoc

```diff
@@ -137,6 +141,50 @@ image::manage-xdcr/xdcr-advanced-settings.png[,400,align=left]
 The values displayed in the fields are defaults, which can be modified interactively, and saved: this may help in achieving optimal replication-performance.
 For details on the significance of each field, see the xref:xdcr-reference:xdcr-reference-intro.adoc[XDCR Reference].
 
+[#identify-target-cluster-in-xdcr]
+=== Identify the Target Cluster in a Replication
+
+A cluster is identified as a target cluster in XDCR when it receives replicated data from a remote cluster.
+Then the replicating remote cluster is identified as a source cluster.
+This is recognized as target awareness and is used to display incoming replications on the target cluster.
```

Comment:
Remove the sentence on line 149 -- this sentence is not needed.   "This is recognized as target awareness and is used to display incoming replications on the target cluster" 

There is no industry terminology, and target awareness was just an internal terminology for describing the feature.  The documentation just needs to explain what incoming replications info shows and why the customer may want to see this info.